source/_posts/routing-in-react.md

```diff
@@ -0,0 +1,590 @@
+---
+title: Routing in React
+contributor: Saurabh Ghatnekar
+avatar: saurabh-ghatnekar.png
+description: Learn how to use React Router to create a single-page application with multiple routes.
+date: 2023-02-17
+tags: [react, routing]
+id: routing-in-react
+relatedPost: [react, routing]
+---
+{% image_fw 1.78 banner.png "Routing in React" %}
+
+# Routing in React
+
+## Table of Contents
+<!-- toc -->
+
+## Introduction
+
+
+Routing in React JS is a way for a computer program to help you move between different pages or sections in a reactjs
```

Comment:
Please be consistent in using formal names. `React JS` or `reactjs`
- [ ] Use `react.js` as this is an official name
- [ ] remove the new line!